Resources/Locale/en-US/crew-medals/crew-medal-component.ftl

```diff
@@ -0,0 +1,22 @@
+# interaction
+comp-crew-medal-inspection-text = Awarded to {$recipient} for {$reason}.
+comp-crew-medal-award-text = {$recipient} has been awarded the {$medal}.
+
+# round end screen
+
+comp-crew-medal-round-end-result = {$count ->
+    [one] There was one medal awarded:
+    *[other] There were {$count} medals awarded:
+}
+
+comp-crew-medal-round-end-list =
+    - [color=white]{$recipient}[/color] earned the [color=white]{$medal}[/color] for
+    {"  "}{$reason}
```

Comment:
This allows users to put markup into round end screen, so a player could award a medal for "[head=1][color=#ff0000]OMGHAX" and really mess up the window.